schema.graphql

```diff
@@ -544,6 +544,7 @@ type Mutation {
   unlikeComment(id: ID!): Comment
   unlikePost(id: ID!): Post
   unregisterForEventByUser(id: ID!): Event!
+  updateAdvertisement(data: UpdateAdvertisementData!, id: ID!): Advertisement
```

Comment:
consolidate the data requirements of a mutation within single variable named `input`, take a look at `UpdateUserTagInput`

Reply:
@xoldyckk updated the code with the changes